chrome/content/zotero/bindings/itembox.xml

```diff
@@ -1975,8 +1978,19 @@
 						default:
 							throw ("Invalid transform mode '" + mode + "' in zoteroitembox.textTransform()");
 					}
-					this._setFieldValue(label, newVal);
-					this._modifyField(label.getAttribute('fieldname'), newVal, this.saveOnEdit);
+					
+					var currentFieldName = label.getAttribute('fieldname');
+					if (!currentFieldName.contains('-')) {// single field, e.g. title
```

Comment:
```String.prototype.contains``` isn't actually valid after Firefox 36 — it's [switching to String.prototype.includes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes). Have to just use ```indexOf``` for the time being.